### PR TITLE
fix: remove accumulated polling when app comes back to foreground

### PR DESCRIPTION
**Pull request by jaeopt**
* When app goes to background, polling stops OK as expected.
* But when app comes back to foreground, the current implementation looks at the timer behind and try to poll multiple times, which is annoying. This fix cleans up this logic, so recover normal periodic polling regardless of delayed timer value.

### CODE REVIEW

File: Sources/Implementation/DefaultDatafileHandler.swift [hunk 1]
```diff
@@ -159,19 +159,6 @@ class DefaultDatafileHandler: OPTDatafileHandler {
         }
     }
     
-    @objc
```
Comment (thomaszurkan-optimizely) [inline Sources/Implementation/DefaultDatafileHandler.swift]: why was this removed?

Comment (jaeopt) [reply Sources/Implementation/DefaultDatafileHandler.swift]: It is not used anywhere. probably leftover from previous changes.

Comment (thomaszurkan-optimizely) [reply Sources/Implementation/DefaultDatafileHandler.swift]: leftover from ios 9 support. got it. :)
